openage/opus/opustest.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/python3
+
+import opusenc
+import time
+import sys
+
+def convert():
+    wavname = "Pieces.wav"
+    verbose = False
+    for arg in sys.argv[1:]:
+        if arg in ("-v", "v"):
+            verbose = True
+        elif arg in ("-nv", "n", "-n"):
+            verbose = False
+        else:
+            wavname = arg
+
+    if verbose:
+        print("Opening wave-file {}".format(wavname))
+
+    wav = open(wavname, mode='rb').read()
+
+    if verbose:
+        print("Encoding...")
+
+    t = time.time()
+    out = opusenc.encode(wav, verbose)
+    t = time.time() - t
+    if type(out) in (str, int):
+        print("Error: {}".format(out))
+        return 1
+
+    outname = wavname.replace("wav", "opus") if wavname.endswith("wav") else wavname + ".opus"
+    opus = open(outname, mode='wb')
+    opus.write(out)
+    print("Wave size: {}\tOpus size:{}\tcompression: {:.2n}\ttime {}".format(len(wav),len(out),len(out)/len(wav),t))
+
+if __name__ == "__main__":
+    convert()
```

Comment:
might be good if you converted this file into a demo, then you can launch it with `./run test -d demoname`.